spread.yaml

```diff
@@ -3,9 +3,6 @@ project: snapd
 environment:
     GOHOME: /home/gopath
     GOPATH: $GOHOME
-    # TODO: some distros started shipping Go 1.16 which defaults to having
-    # modules enabled, drop this when snapd starts supporting Go modules
-    GO111MODULE: off
```

Comment:
Actually, I think we should try `GO111MODULE: "off"` here, to rule out any implicit boot conversion.

Reply:
Nice, thank works well, thanks for the advice